frontend/viewer/src/project/browse/EntriesList.svelte

```diff
@@ -62,6 +77,15 @@
     if (!entry) return;
     onSelectEntry(entry);
   }
+
+  async function handleDeleteEntry(entry: IEntry) {
+    const headword = writingSystemService.headword(entry);
+    const entryId = entry.id;
+    if (!await dialogsService.promptDelete($t`Entry: ${headword}`)) return;
+    await miniLcmApi.deleteEntry(entryId);
+    projectEventBus.notifyEntryDeleted(entryId);
+    if (selectedEntry?.id === entryId) onSelectEntry(undefined);
```

Comment:
I think this last line would be slightly better located in the onEntryDeleted event handler.